modules/frontend/server.py

```diff
@@ -6,6 +6,10 @@
 
 @app.route("/", methods=['GET'])
 def index():
+    return render_template('home.html')
+
+@app.route("/png-extraction", methods = ['GET'])
+def PNG_Extraction():
```

Comment:
can you name it,
"extract_png() to follow the style of the rest of the functions in the module?

Reply:
Ok